content/en/docs/usage/gateway-api.md

```diff
@@ -0,0 +1,332 @@
+---
+title: "Securing Gateway Resources"
+linkTitle: "Securing Gateway Resources"
+weight: 105
+type: "docs"
+---
+
+Since 1.5, cert-manager supports requesting TLS certificates using annotations
+on Gateway resources. This works similarly as to what you can do with
+annotations on the Ingress resource, as described on the page [Securing Ingress
+Resources](/docs/usage/ingress/).
```

Comment:
```suggestion
 **FEATURE STATE**: cert-manager v1.5 [stable] 

cert-manager can generate TLS certificates for Gateway resources.
This is configured by adding annotations to a Gateway and is similar to the process for [Securing Ingress
Resources](/docs/usage/ingress/).
```

Reply:
I like your idea!

I'll go with 1.5 instead of v1.5 as per https://cert-manager.io/docs/installation/supported-releases/#terminology (1.5 is the release, v1.5 is a tag)

```md
**FEATURE STATE**: cert-manager 1.5 [stable]
                                ^^^
```

Reply:
(I re-read the "Terminology" section and it is quite confusing... like release vs. version, patch release vs. patch release...)